### PR TITLE
Amcollie 766 jmartin followon

### PR DESCRIPTION
**Pull request by jmartin-sul**
a couple small touchups that were as easy to make as they were to comment on, so PR to the PR, for your convenience if you like the suggestions

spawned by #833

### CODE REVIEW

File: src/components/editor/ResourceURIMessage.jsx [hunk 1]
```diff
@@ -1,24 +1,20 @@
...
+// Copyright 2019 Stanford University see LICENSE for license
 
 import React from 'react'
 import { connect } from 'react-redux'
 import PropTypes from 'prop-types'
 
 // Renders the resource URI message after Save & Publish
 const ResourceURIMessage = (props) => {
-  if (!props.show) {
+  if (!props.show || !props.uri) {
```
Comment (jmartin-sul) [inline src/components/editor/ResourceURIMessage.jsx]: ~~stockholm~~ rubocop syndrome has me looking for places to use guard clauses

File: src/components/editor/ResourceTemplate.jsx [hunk 2]
```diff
@@ -7,7 +7,7 @@ import ResourceTemplateForm from './ResourceTemplateForm'
 import { rootResourceTemplateLoaded } from 'actions/index'
 import { getResourceTemplate } from 'sinopiaServer'
 import { rootResource } from 'selectors/resourceSelectors'
-import ResourceURIMessage from './resourceURIMessage'
+import ResourceURIMessage from './ResourceURIMessage'
```
Comment (ndushay) [inline src/components/editor/ResourceTemplate.jsx]: wow - this worked with the wrong case?  my world is broken.

File: src/actions/index.js [hunk 3]
```diff
@@ -44,9 +44,9 @@ export const assignBaseURL = item => ({
   payload: item,
 })
 
-export const showResourceURIMessage = show => ({
+export const showResourceURIMessage = resourceUri => ({
```
Comment (ndushay) [inline src/actions/index.js]: I like this change.

File: src/components/editor/ResourceURIMessage.jsx (hunk 1 above)
Comment (ndushay) [reply src/components/editor/ResourceURIMessage.jsx]: I like this change.  And the humor.

File: src/components/editor/ResourceTemplate.jsx (hunk 2 above)
Comment (jmartin-sul) [reply src/components/editor/ResourceTemplate.jsx]: ah, no, i renamed the file too (that was my first change).

actually, i think it did work without the case change, but i did it the other way (renamed the file, ran the tests successfully iirc, got called out by eslint on the import when circle ran the build).

File: src/actions/index.js (hunk 3 above)
Comment (jmartin-sul) [reply src/actions/index.js]: thx!

File: src/components/editor/ResourceURIMessage.jsx (hunk 1 above)
Comment (aaron-collier) [reply src/components/editor/ResourceURIMessage.jsx]: I like this change. I started here, as my mind is likewise in ruby guard clauses, but didn't see it often in examples...

**Pull request merged**